controllers/administration.js

```diff
@@ -846,23 +846,23 @@ router.all('/teachers', permissionsHelper.permissionsChecker(['ADMIN_VIEW', 'TEA
             itemsPerPage = filterQuery["$limit"];
         }
     }
-
+    
     const currentPage = parseInt(req.query.p) || 1;
     let title = returnAdminPrefix(res.locals.currentUser.roles);
-
+    
```

Comment:
Looks like there is something wrong with your editor settings. Empty Lines shoudn't have indentations at all.